### PR TITLE
The new way to include create button in content toolbar section

### PR DESCRIPTION
**Pull request by naderjavid**
Hello, I check last version of abp framework and they are add "Create" button in content toolbar

### CODE REVIEW

**Comment by gdlcf88** [comment]
What is the "content toolbar" and why do we need it? Can you explain? :)

**Comment by naderjavid** [comment]
The new version of abp framework uses this content toolbar for top buttons in all modules
please check these screenshots and you find out about the content toolbar
and also i checked all the abp modules and all of them has this code
@section content_toolbar {
    @await Component.InvokeAsync(typeof(AbpPageToolbarViewComponent), new { pageName = typeof(IndexModel).FullName })
}

**Comment by naderjavid** [comment]
This is a example in the main repository
[https://github.com/abpframework/abp/blob/dev/modules/identity/src/Volo.Abp.Identity.Web/Pages/Identity/Users/Index.cshtml]

File: src/AbpHelper.Core/Commands/Ef/Migrations/Add/AddCommand.cs
Comment (naderjavid) [inline src/AbpHelper.Core/Commands/Ef/Migrations/Add/AddCommand.cs:1]: including Startup Project in command cause error

**Pull request closed without merging**